.circleci/config.yml

```diff
@@ -47,6 +47,7 @@ jobs:
                 conda init powershell
                 conda update conda
                 conda create -n py37 python=3.7 pytorch --yes
+                git config --system core.longpaths true
```

Comment:
is this required ? If so, why not have it in run_dataset_script_tests_pyarrow_5_WIN as well ? If not, feel free to remove it

Reply:
It was required for me to install from my github branch during debugging, but it won't be required when installing bigbench from the current tar file (and when we update to the proper pip install). I will remove it.

I really don't understand exactly what is going on with the dependencies, and it seems rather flaky. 
The only thing that's changed between [e356fb6](https://github.com/huggingface/datasets/pull/4125/commits/e356fb66f8255f1752743c2bd5e3ae26a3167515) and the most recent commit [db51cd2](https://github.com/huggingface/datasets/pull/4125/commits/db51cd2df079f15e1e43cf55b184ac3b7655a915) is the addition of explicitly requiring `sacremoses`.
However, if you look back at the tests that passed on commit [890a49f](https://github.com/huggingface/datasets/pull/4125/commits/890a49f52db6c1fe376cd0f3f537d5a8dd4d8430) it installed `sacremoses` without being explicitly listed. 

The bigbench dependencies have remained fixed between [e356fb6](https://github.com/huggingface/datasets/pull/4125/commits/e356fb66f8255f1752743c2bd5e3ae26a3167515) and  [db51cd2](https://github.com/huggingface/datasets/pull/4125/commits/db51cd2df079f15e1e43cf55b184ac3b7655a915), so there must be some other external version update that's suddenly making pip struggle with the windows install.

One observation I made in tracking down the issue is that in the pip install log for the jobs that time out is that it's trying to install the latest `pyarrow` version (version 8.0.0) which is incompatible with the most recent `apache-beam v2.38`. `apache-beam 2.38.0` requires `pyarrow<7.0.0,>=0.15.1`
There is a new `apache-beam v2.39rc2` out as of yesterday which should be made official in the next few days. This is compatible with pyarrow 8.0.0, so maybe that will fix the problem? 
I will try to pin `apache-beam==2.39rc2` as an experiment and remove `sacremoses` to see if this is the issue.

Reply:
We had to add sacremoses a few days go as well, but it's unrelated to your PR: https://github.com/huggingface/datasets/commit/1a4c185663a6958f48ec69624473fdc154a36a9d

Reply:
Can you merge upstream/master into your branch and remove the `sacremoses` you added ? We've already added it here:

https://github.com/huggingface/datasets/blob/c1dce72ff259d3e7abee34900e48578ec685fb00/setup.py#L147-L155

The rest looks all good, I think we can merge this PR after this ! Is there anything else you wanted to do before we merge this PR ?